pulsar-broker/src/main/java/org/apache/pulsar/broker/lookup/TopicLookupBase.java

```diff
@@ -79,6 +79,7 @@ protected CompletableFuture<LookupData> internalLookupTopicAsync(final TopicName
                 })
                 .thenCompose(exist -> {
                     if (!exist) {
+                        pulsar().getBrokerService().getLookupRequestSemaphore().release();
```

Comment:
A better way to fix this is moving `exceptionally`(line-135) to line-139. WDYT?

Reply:
thx to review. It is a good idea, i will recommit it.

Reply:
> A better way to fix this is moving `exceptionally`(line-135) to line-139. WDYT?

thx to review. It is a good idea, i will recommit it.